__tests__/collection.test.ts

```diff
@@ -435,3 +435,35 @@ describe('random thisArg tests', () => {
 		}, array);
 	});
 });
+
+describe('ensure() tests', () => {
+	function createTestCollection() {
+		return new Collection([
+			['a', 1],
+			['b', 2],
+		]);
+	}
+
+	test('set new value if key does not exist', () => {
+		const coll = createTestCollection();
+		coll.ensure('c', () => 3);
+		expect(coll.size).toStrictEqual(3);
+		expect(coll.get('c')).toStrictEqual(3);
+	});
+
+	test('return existing value if key exists', () => {
+		const coll = createTestCollection();
+		const ensureB = coll.ensure('b', () => 3);
+		const getB = coll.get('b');
+		expect(ensureB).toStrictEqual(2);
+		expect(getB).toStrictEqual(2);
+		expect(coll.size).toStrictEqual(2);
+	});
+
+	test('ensure with existing key should not change the collection', () => {
+		const coll = createTestCollection();
+		coll.ensure('a', () => 4);
+		expect(coll.size).toStrictEqual(2);
+		expect(coll.get('a')).toStrictEqual(1);
+	});
```

Comment:
aren't these two doing exactly the same?

Reply:
Yes, they are.

The second one should be removed, as the first one checks for ensure's returned value.